test/Stubs.scala

```diff
@@ -0,0 +1,111 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.typesafe.config.{Config, ConfigFactory}
+import play.api.Configuration
+import play.api.http.{DefaultFileMimeTypes, FileMimeTypes, FileMimeTypesConfiguration}
+import play.api.i18n.{Langs, MessagesApi}
+import play.api.mvc._
+import play.api.test.Helpers._
+import play.api.test.NoMaterializer
+import uk.gov.hmrc.govukfrontend.views.html.components
+import uk.gov.hmrc.govukfrontend.views.html.components.{GovukHeader, Footer => _, _}
+import uk.gov.hmrc.hmrcfrontend.config.{AccessibilityStatementConfig, TrackingConsentConfig}
+import uk.gov.hmrc.hmrcfrontend.views.html.components.{HmrcBanner, HmrcFooter, HmrcHeader, HmrcReportTechnicalIssue}
+import uk.gov.hmrc.hmrcfrontend.views.html.helpers.{HmrcFooterItems, HmrcTrackingConsentSnippet, hmrcStandardFooter}
+import uk.gov.hmrc.plasticpackagingtax.registration.config.AppConfig
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.partials.{phaseBanner, siteHeader}
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import scala.concurrent.ExecutionContext
+
+trait Stubs {
+
+  def stubMessagesControllerComponents(
+    bodyParser: BodyParser[AnyContent] = stubBodyParser(AnyContentAsEmpty),
+    playBodyParsers: PlayBodyParsers = stubPlayBodyParsers(NoMaterializer),
+    messagesApi: MessagesApi = stubMessagesApi(),
+    langs: Langs = stubLangs(),
+    fileMimeTypes: FileMimeTypes = new DefaultFileMimeTypes(FileMimeTypesConfiguration()),
+    executionContext: ExecutionContext = ExecutionContext.global
+  ): MessagesControllerComponents =
+    DefaultMessagesControllerComponents(
+      new DefaultMessagesActionBuilderImpl(bodyParser, messagesApi)(executionContext),
+      DefaultActionBuilder(bodyParser)(executionContext),
+      playBodyParsers,
+      messagesApi,
+      langs,
+      fileMimeTypes,
+      executionContext
+    )
+
+  private val minimalConfig: Config = ConfigFactory.parseString("""
+      |assets.url="localhost"
+      |assets.version="version"
+      |google-analytics.token=N/A
+      |google-analytics.host=localhostGoogle
+      |tracking-consent-frontend.gtm.container=a
+      |metrics.name=""
+      |metrics.rateUnit="SECONDS"
+      |metrics.durationUnit="SECONDS"
+      |metrics.showSamples=false
+      |metrics.jvm=false
+      |metrics.logback=false
+      |draft.timeToLive=1d
+      |timeoutDialog.timeout=13min
+      |timeoutDialog.countdown=3min
+      |list-of-available-journeys = "SMP,STD,CAN,SUB,CON"
+      |microservice.services.features.default=disabled
+      |microservice.services.features.use-improved-error-messages=true
+      |urls.tradeTariff=tradeTariff
+      |urls.classificationHelp=classificationHelp
+      |urls.ecicsTool=ecicsTool
```

Comment:
We should not need these Config options, as these are just specific to Exports

Reply:
removed